lib/assets/javascripts/set_builder.js

```diff
@@ -527,3 +531,34 @@ Array.prototype.__find = function(fn) {
   return null;
 }
 if(!Array.prototype.find) Array.prototype.find = Array.prototype.__find;
+
+//
+// .select
+//
+Array.prototype.__select = function(fn) {
+  var results = [];
+  for(var i=0; i<this.length; i++) {
+    if(fn(this[i])) {
+      results.push(this[i]);
+    }
+  }
+  return results;
+}
+if(!Array.prototype.select) Array.prototype.select = Array.prototype.__select;
+
+
+//
+// .any
+//
+Array.prototype.__any = function(functionOrValue) {
+  var results;
+  if(typeof(functionOrValue) == "function") {
+    results = this.__select(functionOrValue);
+  } else {
+    results = this.__select(function(value) {
+      return value === functionOrValue;
+    });
+  }
+  return results.length > 0;
+}
+if(!Array.prototype.any) Array.prototype.any = Array.prototype.__any;
```

Comment:
Hey Ben, I _don't think_ you ended up using this `any` method. Does that sound right? If so you could remove it and the tests for it.